admin/settings/class-admin-apple-settings-section.php

```diff
@@ -582,13 +582,22 @@ public function render_field( $args ) {
 			$field .= apply_filters( 'apple_news_field_description_output_html', '<br/><i>' . $description . '</i>', $name );
 		}
 
-		return sprintf(
-			$field,
-			esc_attr( $name ),
-			esc_attr( $value ),
-			intval( $size ),
-			$this->is_required( $name )
-		);
+		// Use the proper template to build the field
+		if ( is_array( $type ) || 'font' === $type || 'boolean' === $type ) {
+			return sprintf(
+				$field,
+				esc_attr( $name )
+			);
+		} else {
+			return sprintf(
+				$field,
+				esc_attr( $name ),
+				esc_attr( $value ),
+				intval( $size ),
+				$this->is_required( $name )
```

Comment:
Even though it can only be `'required'` or `''`, maybe late-escape anyway for consistency?